### PR TITLE
Design mobail varshion hero section

### PR DESCRIPTION
**Pull request by jubaer919**
added some necessary file for microverse linter and added html and css file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,700;1,300;1,400&display=swap" rel="stylesheet"> 
+    <link rel="stylesheet" href="style.css">
+    <title>Jubaer Maruf</title>
+</head>
+<body>
+    <header class="header">
+        <nav class="nav">
+            <h2 class="logo">My Logo</h2>
```
Comment (AlphaNtihinduka) [inline index.html]: - [ ] If you notice most websites have a logo, which when clicked on, the page is redirected to the home page ☺️. Therefore, i suggest that you ensure that the logo is enclosed in `a` tag and also removed from inside the `nav` tag. This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            <img src="./images/stick.svg" alt="logo of stick" class="stick">
+            <img src="./images/half-circle.svg" alt="" class="bottom-half-circle">
+            <img src="./images/half-ring.svg" alt="logo of half ring" class="bottom-half-ring">
+            <img src="./images/ring.svg" alt="logo of ring" class="bottom-ring">
+            <img src="./images/full-circle.svg" alt="logo of circle" class="top-full-circle">
+            <img src="./images/stick.svg" alt="logo of stick" class="top-stick">
+            <img src="./images/half-circle.svg" alt="" class="top-half-circle">
+            <img src="./images/big-stick.svg" alt="logo of stick" class="top-left-stick">
+            <img src="./images/big-stick.svg" alt="logo" class="big-stick-right">
+            <img src="./images/big-ring.svg" alt="logo" class="big-ring-right">
```
Comment (AlphaNtihinduka) [inline index.html]: - [ ] You have done a great job to ensure that there is a background image appearing. However, you should know that this is for design purposes as we try to make our portfolio look nice. Therefore the background image should be added using `CSS` styles, not as an image ☺️.
Keynote: Kindly ensure to use one background image in CSS for easy formating.

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @jubaer919 🙋‍♂️
Good job working on your mobile setup. You have done the design to the best expectation and incredibly well. 👏  

You still need to implement few on the project but you are almost there!!! 🚶: 

TO HIGHLIGHT:
- Your project is professional ✅
- Professional README file :100: 
- Your PR is professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

# Required Changes :recycle: 
- Kindly consider the inline comments.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

**Review by joel-kalema — APPROVED** [review]
Hi @jubaer919 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing 👍 
The project is running successfully 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**